fwoa-tools/src/binaryConverter.ts

```diff
@@ -115,10 +101,17 @@ async function convertBinaryResource(): Promise<void> {
     return;
   }
   logs.write(`${new Date().toISOString()}: Retrieved All Binary Keys from migration export output.\n`);
+  let tenantPrefix = '';
```

Comment:
Is this an actual functional change? Do we need to re-run the tests on binary resources for this change?